contracts/0.8.9/WithdrawalQueue.sol

```diff
@@ -104,8 +104,18 @@ abstract contract WithdrawalQueue is AccessControlEnumerable, PausableUntil, Wit
 
     /// @notice Pause withdrawal requests placement and finalization. Claiming finalized requests will still be available
     /// @param _duration pause duration, seconds (use `PAUSE_INFINITELY` for unlimited)
+    /// @dev Reverts with `ResumedExpected()` if contract is already paused
+    /// @dev Reverts with `AccessControl:...` reason if sender has no `PAUSE_ROLE`
+    /// @dev Reverts with `ZeroPauseDuration()` if zero duration is passed
     function pause(uint256 _duration) external onlyRole(PAUSE_ROLE) {
-        _pause(_duration);
+        _pauseFor(_duration);
+    }
+
+    /// @notice Pause withdrawal requests placement and finalization. Claiming finalized requests will still be available
+    /// @param _resumeSince the first second to resume since
+    /// @dev Reverts with `ResumeSinceInPast()` if the timestamp is in the past
+    function pauseUntil(uint256 _resumeSince) external onlyRole(PAUSE_ROLE) {
```

Comment:
Every time I read "until" anywhere I'm not sure if the right boundary is supposed to be included. Since sounds less ambiguous for me. But I might not getting the language subtlety right. Any other opinions?

Reply:
Yeah, both are correct https://ell.stackexchange.com/questions/33340/is-until-inclusive-or-exclusive

Shall we describe the meaning in the comment and proceed with the inclusive variant?

Reply:
yep, done